react/Field/index.jsx

```diff
@@ -67,25 +124,29 @@ const Field = props => {
 }
 
 Field.PropTypes = {
+  enableExtraFunction: PropTypes.bool,
   fullwidth: PropTypes.bool,
   label: PropTypes.string.isRequired,
   id: PropTypes.string,
   type: PropTypes.oneOf(['text', 'password', 'email', 'url', 'textarea']),
   value: PropTypes.string,
   placeholder: PropTypes.string,
   error: PropTypes.bool,
-  size: PropTypes.oneOf(['tiny', 'medium', 'large'])
+  size: PropTypes.oneOf(['tiny', 'medium', 'large']),
+  secondaryLabels: PropTypes.object
 }
 
 Field.defaultProps = {
+  enableExtraFunction: true,
```

Comment:
This name is not very explicit and can be confusing as we can expect a function here.

After discussing with @ptbrowne, the best way to achieve this seems to pass `showVisibilityButton` inside a `fieldProps` prop.